reactive/kotlinx-coroutines-reactive/src/Publish.kt

```diff
@@ -17,18 +17,15 @@ import kotlin.internal.LowPriorityInOverloadResolution
 
 /**
  * Creates cold reactive [Publisher] that runs a given [block] in a coroutine.
- * Every time the returned publisher is subscribed, it starts a new coroutine.
- * Coroutine emits items with `send`. Unsubscribing cancels running coroutine.
+ * Every time the returned flux is subscribed, it starts a new coroutine in the specified [context].
+ * Coroutine [emits][Subscriber.onNext] values with `send`, [completes][Subscriber.onComplete]
```

Comment:
The names of subscriber methods (`onNext`, `onComplete`, and `onError`) are now hidden from the readers of docs (you can only see them if you click a link), while "emits", "completes", and "emits error" terminology might not be truly idiomatic for reactive world. I'd suggest to give them in parenthesis like this:

"Coroutines emits (`onNext`) values with `send` ...."

(Same applies to `flux`, `mono`, and `rxXxx` builders)

Reply:
Made it clear in multi-value builders, left no references at all in single-value (mono, single, etc.) because it's pretty straightforward